templates/VERSION.sh.template

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 gRPC authors.
```

Comment:
This should be located at `templates/internal_ci/linux/VERSION.sh`. I'm wondering how the templated file ended up in the proper place. Was that a manual move?

Reply:
Yes, I created it (`VERSION.sh` file) manually just to be able to test this code from kokoro. Thanks for pointing out the problem. BTW, there might be an issue with this approach - let's discuss offline.

Reply:
BTW, if this is to mirror the rest of the repo, shouldn't it be `templates/tools/internal_ci/linux/VERSION.sh.template` or is `tools` assumed?

Reply:
Yes, it should be under tools. I was just copying from the path of your manually moved file below.

Reply:
All the VERSION stuff has been removed